sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1663,7 +1663,7 @@ def test_ridge_fit_intercept_sparse(solver, with_sample_weight, global_random_se
     sparse_ridge.fit(sp.csr_matrix(X), y, sample_weight=sample_weight)
 
     assert_allclose(dense_ridge.intercept_, sparse_ridge.intercept_)
-    assert_allclose(dense_ridge.coef_, sparse_ridge.coef_)
+    assert_allclose(dense_ridge.coef_, sparse_ridge.coef_, rtol=5e-7)
```

Comment:
Was this change required for `test_ridge_fit_intercept_sparse` to locally pass for you?

Reply:
Yes, to pass for all random seeds.